gateway/resolver/arguments.go

```diff
@@ -160,3 +170,36 @@ func validateSortBy(items []unstructured.Unstructured, fieldPath string) error {
 
 	return nil
 }
+
+func getDryRunArg(args map[string]interface{}, key string, required bool) ([]string, error) {
```

Comment:
Since we have one value that is "All", lets make dryRun type as bool and we can skip this complicated processing.

Reply:
I don't see the change, let me clarify it more:
I want to pass it like this:
```
mutation {
  core {
    deletePod(name: "my-new-pod", namespace: "default", dryRun: true)
  }
}
```

And use already implemented getBoolArg to get it.